components/TARGET_PSA/services/crypto/COMPONENT_PSA_SRV_IPC/crypto_types.h

```diff
@@ -0,0 +1,5 @@
+#ifdef PSA_CRYPTO_SECURE
```

Comment:
license header files in each file

Reply:
@0xc0170 these files were introduced in PR #9575, the commits relevant to this PR start from c913042. I've added the license in #9575 (3bc5f9086c3bbf1471906d6a4753457cf46f19cf), this branch will be rebased once 9575 goes through.